assets/apps/dashboard/src/Components/LicenseCard.js

```diff
@@ -66,12 +70,23 @@ const LicenseCard = ({ isVisible, setSettings, changeLicense, license }) => {
 		<aside className="sidebar card license">
 			<div className="sidebar-section">
 				{licenseCardHeading && <h4>{licenseCardHeading}</h4>}
-				{!whiteLabel && licenseCardDescription && (
-					<p
-						dangerouslySetInnerHTML={{
-							__html: licenseCardDescription,
-						}}
-					/>
+				{!whiteLabel && (
+					<p>
+						{typeof createInterpolateElement !== 'undefined' &&
+							createInterpolateElement(
```

Comment:
@UVLabs I don't think we need the `createInterpolateElement.` Have a look at https://stackoverflow.com/questions/610406/javascript-equivalent-to-printf-string-format.
However, now that I'm seeing it, moving this to JS might add transition changes so I think the first solution you provided was better.
You can try https://stackoverflow.com/questions/610406/javascript-equivalent-to-printf-string-format and if it adds translation issues, let's get back to your initial solution with the remark that the icon should have the same color as the link.
Sorry for misleading you.

Reply:
Thank you for changing it again! I know is different but I don't want to add any translation changes for this.